### PR TITLE
startup server if it's not running

### PR DESCRIPTION
**Pull request by simylein**
My main idea would be if someone has the restart script running daily as an example and the server goes down it would startup automatically on the next execution of restart.sh instead of telling you that there is no server running. 
Also I combined some countdown messages. 
I think the three install commands in README.md, they could be in a code box because they are rather essential for users.
Feel free to edit or change it to suit your needs ;-)
FYI I noticed one could provide a single line of code that downloads, makes executable and executes your SetupMinecraft.sh script. 
I added an update script on request of @capsload2 which updates to the latest version available on papermc.io.

### CODE REVIEW

File: restart.sh [hunk 1]
```diff
@@ -7,28 +7,24 @@
 # Check if server is running
 if ! screen -list | grep -q "\.minecraft"; then
     echo "Server is not currently running!"
-    exit 1
+    echo "Starting up server now..."
+    ./start.sh
```
Comment (Minecraftschurli) [inline restart.sh:11]: Why start the server before restarting

File: update.sh [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+# execute stop script
+./stop.sh
+
+# grep latest verion from papermc.io
+Version=$(wget -q -O - https://papermc.io/api/v2/projects/paper | rev | cut -d, -f1 | rev)
+Version="${Version:1}"
+Version="${Version::-3}"
+echo "latest server version seems to be $Version"
+echo "trying to update to version $Version ..."
```
Comment (Minecraftschurli) [inline update.sh]: Maybe prompt the user if they want to update

File: restart.sh (hunk 1 above)
Comment (simylein) [reply restart.sh:11]: You are absolutely right, that is a logical error. I meant to exit after the startup. Thanks for noticing :-)

File: update.sh (hunk 2 above)
Comment (simylein) [reply update.sh]: Well, personally I prefer user input, but some people want a "it just works" solution. I don't know what people prefer more. Of course I can build in a confirmation prompt.

**Comment by TheRemote** [comment]
Hey guys,

I love it!  The only tricky thing about automatic updates with the Java version of Minecraft is they are often really unstable and have issues.  The 1.17 launch has been pretty smooth so far but the last several were anything but!

I'd probably be totally open to implementing something like this if it was an option during SetupMinecraft.sh!

**Pull request closed without merging**